app/api/health-check/route.ts

```diff
@@ -22,7 +22,11 @@ export async function POST(request: Request) {
     const { instanceId } = healthCheckSchema.parse(body);
     
     try {
-      const healthResult = await checkInstanceHealth(instanceId);
+      const version = getAgentVersion();
+      const healthResult = await checkInstanceHealth(instanceId, "us-east-1", {
+        environment: 'production',
```

Comment:
Nuke this arg altogether. The health check should be able to look up the version internally